src/widget/textbox.rs

```diff
@@ -386,8 +398,15 @@ impl Widget<String> for TextBox {
                         self.backspace(data);
                         self.reset_cursor_blink(ctx);
                     }
-                    // Actual typing
-                    k_e if k_e.key_code.is_printable() => {
+                    // Delete
+                    k_e if (HotKey::new(None, KeyCode::Delete)).matches(k_e) => {
+                        self.delete(data);
+                        self.reset_cursor_blink(ctx);
+                    }
+                    // Actual typing (when Ctrl or Alt or Meta is not held down)
+                    k_e if k_e.key_code.is_printable()
+                        && !(k_e.mods.alt || k_e.mods.ctrl || k_e.mods.meta) =>
```

Comment:
On some platforms (macOS) alt produces output that we expect to see in the text field; for instance it is how you would type åççéñ†é∂ and alternate characters.